source/features/transactions/components/TransactionInfo.tsx

```diff
@@ -0,0 +1,102 @@
+import { observer } from 'mobx-react-lite';
+import moment from 'moment';
+import React from 'react';
+import DividerWithTitle from '../../widgets/divider-with-title/components/DividerWithTitle';
+import styles from './TransactionInfo.scss';
+
+export interface ITransactionInfoProps {
+  id: string;
+  currentAddress: string;
+  transferredAt: number;
+  senders: Array<string>;
+  receivers: Array<string>;
+  amounts: Array<number>;
+}
+
+const ArrowNext = require('../../../static/assets/images/arrow-next.svg');
+const SEVEN_DAYS = 7 * 24 * 3600000;
+
+const shrinkAddress = (address: string) => {
```

Comment:
In case you need to use this function which was already defined in transaction list, it's recommended to move this function to more common level such as `helpers` and reuse it here and other places.

Reply:
Added common/helpers.ts where i moved this shrinkAddress function